samples/http_callback.js

```diff
@@ -0,0 +1,22 @@
+import http from "k6/http"
+
+globalThis.requestP = (method, url, body, params) => {
+  return new Promise((resolve, reject) => {
+    params = params || {};
+    params.callback = (resp, error) => {
+      if (error != null) {
+        reject(error)
+      } else {
+        resolve(resp)
+      }
+    }
+    http.request(method, url, body, params)
+  })
+}
+
+export default () => {
+  http.request("GET", "https://httpbin.test.k6.io/delay/5", null, { callback: (resp) => { console.log(JSON.stringify(resp, null, "  ")) } })
+  requestP("GET", "https://httpbin.test.k6.io/delay/4").then((resp) => { console.log(resp) })
+  console.log("something")
+
+}
```

Comment:
```suggestion
export default () => {
    http.request("GET", "https://httpbin.test.k6.io/delay/5", null, {
        callback: (resp) => {
            console.log(JSON.stringify(resp, null, "  "))
        }
    })
    requestP("GET", "https://httpbin.test.k6.io/delay/4").then((resp) => {
        console.log(resp)
    })
    console.log("something")
}
```

Reply:
It seems the latest rebase reverted it

Reply:
oops, I guess this is why I had problem rebasing :thinking: . Sorry I have fixed it, thanks for catching this :bow: